theme/imageuploadloader.css

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2003-2018, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+:root {
+	--ck-upload-placeholder-loader-size: 24px;
+}
+
+.ck .ck-upload-placeholder-loader {
+	position: absolute;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	height: 100%;
+	top: 0;
+	left: 0;
+
+	&::before {
+		content: '';
+		position: relative;
+		width: var(--ck-upload-placeholder-loader-size);
```

Comment:
Styles like `width`, `height`, belong to Lark. `position`, `top`, `content`, `flex` should stay here.